salt/modules/flatpak.py

```diff
@@ -0,0 +1,111 @@
+# -*- coding: utf-8 -*-
+'''
+Manage flatpak packages via Salt
+
+.. versionadded:: Neon
+
+:depends: flatpak for distribution
+'''
+
+from __future__ import absolute_import, print_function, unicode_literals
+import subprocess
+import re
+
+import salt.utils.path
+
+FLATPAK_BINARY_NAME = 'flatpak'
+
+__virtualname__ = 'flatpak'
+
+
+def __virtual__():
+    if salt.utils.path.which('flatpak'):
+        return __virtualname__
+
+    return (False, 'The flatpak execution module cannot be loaded: the "flatpak" binary is not in the path.')
+
+
+def install(location, name):
+    '''
+    Install the specified flatpak package or runtime from the specified location.
+    Returns a dictionary of "result" and "output".
+    location
+        The location or remote to install from.
+    name
+        The name of the package or runtime
+    '''
```

Comment:
Please add a `Returns` section.
```
Returns:
  dict: A dictionary showing the status and output of the command
```
Please add a CLI Example showing how you would run this function
```
CLI Example:
  .. code-block:: bash
    salt '*' flatpak.install <whatever goes here> <and here>
```